### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by edumoreira1506**
Hello guys! This PR resolves issue #30 

I followed the 'example', and on code, I followed the same pattern of other components. I didn't add one route for the page, because I think than this is not about this PR, but the component is there, and this is the preview:

Screenshot from 2019-10-03 13-35-41

### CODE REVIEW

File: src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            <div style={{
+              display: 'block',
+              width: '100%',
+              height: '100%'
+            }}>
+              <LoginButton 
+                backgroundColorIcon='#50c94b'
+                backgroundColorText='#45b356'
+                icon={<FontAwesomeIcon icon={faGithub} />}
+                text='Github'
```
Comment (lex111) [inline src/components/Login.js]:
```suggestion
                text='GitHub'
```

File: src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import LoginButton from './LoginButton';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBitbucket, faGithub } from '@fortawesome/free-brands-svg-icons';
+
+const colorBackground = '##e6dbdb';
```
Comment (luanraithz) [inline src/components/Login.js]: Was there supposed to be two `#`?

File: src/components/Login.js (hunk 1 above)
Comment (edumoreira1506) [reply src/components/Login.js]: Oh sure!

File: src/components/Login.js (hunk 2 above)
Comment (edumoreira1506) [reply src/components/Login.js]: Oh, Sorry, this was wrong

**Comment by rafaveira3** [comment]
Hey, all! Thanks a lot for your effort in working on this PR! You rock, guys! 🚀 I have a few comments regarding it! Would you mind also adding:

* A dark background, it could be black or something like that.
* Gitlab instead of BitBucket. We will start our OpenID Connect auth using these two platforms.
* huskyCI logo instead of `Start Building!` string.
* A logic to open this particular page in a `/login` route somehow.

Feel free to work together if you desire! 🙃 I can also help with some commits! What do you think?

File: src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
```
Comment (rafaveira3) [inline src/components/Login.js]: Could you please add `Globo.com` license header in this file? You can find it in other files inside our project.

File: src/components/LoginButton.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```
Comment (rafaveira3) [inline src/components/LoginButton.js]: Could you please add `Globo.com` license header in this file? You can find it in other files inside our project.

File: src/pages/Dashboard.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (rafaveira3) [inline src/pages/Dashboard.js]: Could you please add Globocom license header here?

File: src/pages/Login.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+*/
+
+import React, { Component } from 'react';
+import LoginButton from '../components/LoginButton';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGitlab, faGithub } from '@fortawesome/free-brands-svg-icons';
+import logo from './../logo.png';
+
+const colorBackground = '#171515';
+const colorBackgruondBox = '#3d4572';
```
Comment (rafaveira3) [inline src/pages/Login.js]: Could you fix this typo here, please? `colorBackgruondBox` into `colorBackgroundBox`.

Comment (edumoreira1506) [reply src/pages/Login.js]: Sure!

**Comment by edumoreira1506** [comment]
Conflicts solved!

**Comment by edumoreira1506** [comment]
@rafaveira3 My PR was merged and I opened other PR too, but in hacktoberfest globocom panel is empty :(

Screenshot from 2019-11-12 14-10-59

**Comment by rafaveira3** [comment]
Hey, @edumoreira1506. Could you send me the link to the other PR? I can check it here for you!

**Comment by edumoreira1506** [comment]
@rafaveira3 Sure! It's [here](https://github.com/globocom/huskyCI-dashboard/pull/33)

**Pull request merged**